### DISCUSSION TITLE
CRON Timer?

### DISCUSSION THREAD

**Discussion by MhAllan** [dotnet/csharplang#8286] [Language Ideas]
That is it, how come .net doesn't have cron timer until now?

**Comment by HaloFour** [comment]
That would be more a [runtime](https://github.com/dotnet/runtime) question.  .NET offers numerous timer implementations.  Not sure why you'd want a cron timer given you should probably just use the scheduling facilities offered by the OS in order to run a program periodically vs. having the program always running but doing nothing most of the time.

**Comment by CyrusNajmabadi** [comment]
Not a language question.  Closing out.
